Sources/NIO/EventLoop.swift

```diff
@@ -93,7 +170,6 @@ public protocol EventLoop: EventLoopGroup {
     /// Schedule a `task` that is executed by this `SelectableEventLoop` after the given amount of time.
     func scheduleTask<T>(in: TimeAmount, _ task: @escaping () throws -> T) -> Scheduled<T>
 }
-
```

Comment:
Nit: replace this whitespace line.

Reply:
Can I leave this since it follows the overall formatting where no empty line is before the closing brackets?

Reply:
This line is after the closing brace.